services/121-service/src/payments/fsp-integration/safaricom/dtos/safaricom-transfer-timeout-callback.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty } from 'class-validator';
+
+export class SafaricomTransferTimeoutCallbackDto {
```

Comment:
Do we conceptually call it "Safaricom Timeout Callback" or "Safaricom Transfer Timeout Callback"? The two are now used inconsistently through the code. 

I would opt for: "Safaricom Timeout Callback", and have that besides "Safaricom Transfer Callback", (although strictly speaking "Safaraicom Transfer Timeout Callback" would be more correct, it is easy not to have the word "transfer" twice).